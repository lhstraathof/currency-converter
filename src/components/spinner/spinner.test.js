import { render, screen } from '@testing-library/react';
import { create } from 'react-test-renderer';
import { Spinner } from '@components';

describe('Spinner', () => {
	it('renders correctly', () => {
		render(<Spinner />);
		expect(screen.getByTestId('spinner')).toBeInTheDocument();
	});

	it('matches snapshot', () => {
		const component = create(<Spinner />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
