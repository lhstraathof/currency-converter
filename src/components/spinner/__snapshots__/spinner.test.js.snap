// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner matches snapshot 1`] = `
<div
  className="center"
  data-testid="spinner"
>
  <div
    className="spinner"
  />
</div>
`;
